src/transport.rs

```diff
@@ -16,16 +16,20 @@ fn native_transport(keypair: identity::Keypair) -> Result<Boxed<(PeerId, StreamM
     use std::time::Duration;
 
     use libp2p::{
-        core::upgrade::Version,
+        core::{self, upgrade::Version},
         noise::{self, NoiseConfig, X25519Spec},
         tcp::TcpConfig,
         yamux::YamuxConfig,
         Transport,
     };
+    use libp2p_webrtc::WebRtcTransport;
 
+    let peer_id = PeerId::from(keypair.public());
+    let webrtc = WebRtcTransport::new(peer_id, vec!["stun:stun.l.google.com:19302"]);
     let tcp = TcpConfig::new().nodelay(true);
+    let transport = core::transport::OrTransport::new(webrtc, tcp);
```

Comment:
So the WebRTC transport is enabled even for non wasm builds?

Reply:
webrtc is needed to connect to a browser node and to the cloud-relay which let's us communicate when we aren't in the same local network

Reply:
Exactly, so there can be direct connections native<->browser